app/templates/suppliers/_frameworks_open.html

```diff
@@ -1,41 +1,31 @@
- <div class='grid-row'> 
-  <div class='column-two-thirds'>
+
     {% for framework in frameworks.open %}
       {% if framework.registered_interest %}
-        {%
-          with
-          items = [{
-            "title": "Continue your %s application" | format(framework.name),
-            "link": url_for(".framework_dashboard", framework_slug=framework.slug),
-            "body": framework.deadline,
-          }]
+        {% with
+          headingId = "closed-notice",
```

Comment:
I can't work out what this line does?

Reply:
This looks to me like an old thing that is no longer used in the toolkit anywhere, but that still exists in the example yml files.  I can't see that it ever gets processed anywhere or does anything.

You'd expect it to be included by the macro as the `id` of the heading element generated.  But I'm 99% sure it doesn't actually ever get read.

Reply:
So... I've done a bit of digging.

Those lines were added here: https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/212/files
Along with code in the temporary-message template that aded the ID to the heading in the macro.

But then it got removed from the template here: https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/227

But the examples in the yml files didn't have the `headingId` removed from them.

So we can safely delete `headingId` from all temporary message includes here, and also remove them from the examples in the toolkit.

Reply:
I checked and I am afraid it does not do anything at all. I took it wholesale from pages builder.  I will get rid of it. I will also talk to Chris about taking the line out of pages builder.

Reply:
@Wynndow I will remove this line from supplier-frontend, are you ok to remove it from frontend-toolkit?

Reply:
The same with  `grid = "column-one-third",` line :/